tools/check_test.js

```diff
@@ -30,6 +30,12 @@ function Driver() {
     fail: 0,
     skip: 0,
     timeout: 0,
+    json: {
+      bin: { text:0, total:0, data:0, bss:0, rodata:0 },
```

Comment:
@daeyeon I am not aware. Could you explain more? Is there any issue with design or anything about it?
So until new remote-testrunner will be released I don't need to provide / submit test results for TizenRT? The purpose of this is to gather results when manually running test suite. This is useful for new boards, without test-runner support.